src/v2/view-builder/views/google-authenticator/EnrollGoogleAuthenticatorBarcodeView.js

```diff
@@ -0,0 +1,37 @@
+import { View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+export default View.extend({
+  className: 'oie-enroll-google-authenticator-barcode',
+  template: hbs`
+      {{#if href}}
+      <div class="qrcode-info-title">{{i18n code="oie.enroll.google_authenticator.scanBarcode.title" bundle="login"}}</div>
+      <div class="qrcode-info-container">
+        <p class="qrcode-info">
+          {{i18n code="oie.enroll.google_authenticator.scanBarcode.description" bundle="login"}}
+        </p>
+        <div class="qrcode-container">
+          <img class="qrcode" src={{href}} alt="qr code"></img>
+          <a href="#" class="switch-channel-link">
+            {{i18n code="oie.enroll.google_authenticator.scanBarcode.cannotScan" bundle="login"}}
+          </a>
+        </div>
+        </div>
+      {{/if}}
+    `,
+  initialize () {
+    View.prototype.initialize.apply(this, arguments);
+    this.options.appState.set('isScanBarcode', true);
+  },
+  getTemplateData () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    const isScanBarcode = this.options.appState.get('isScanBarcode');
+    return { href: isScanBarcode && contextualData.qrcode?.href };
+  },
+  postRender () {
+    this.$el.find('.switch-channel-link').on('click', (event) => {
```

Comment:
Don't recommend to put an event handler through DOM.. I suggest to find some alternative way to do it.